ui/raidboss/popup-text.js

```diff
@@ -408,19 +408,18 @@ class PopupText {
       return ValueOrFunction(result['en']);
     };
 
-    let showText = this.options.TextAlertsEnabled;
-    let playSounds = this.options.SoundAlertsEnabled;
-    let playSpeech = this.options.SpokenAlertsEnabled;
-    let playGroupSpeech = this.options.GroupSpokenAlertsEnabled;
+    let SoundOptions = {
+      soundUrl: ValueOrFunction(trigger.sound),
```

Comment:
Yeah, I'll go through again and ensure the ordering. The old PopupText didn't get the ordering right either (a disabled trigger probably shouldn't run its regex against every log line pointlessly, so the disabled check should really be in `ReloadTimelines` instead, otherwise the ordering actually becomes `[id, regex, disabled]`). Do you want me to enforce the ordering up to the `ReloadTimelines` function as well?

Reply:
Agreed that disabled should be in `ReloadTimelines`.

I'm not sure what you mean by enforcing the ordering up to the `ReloadTimelines` function.

Reply:
Meaning, "should I make changes to that function to reflect the stated trigger ordering".